File: Tests/test07/sort_words.pl
#!/usr/bin/perl -w

use strict;

foreach my $line (<STDIN>) {

	$line =~ s/\s+/ /g;

	my @words = split / /, $line;
	
	foreach my $word (sort @words) {
		print "$word ";
	}
	print "\n";
}

File: Tests/test09/sort_words.py
#!/usr/bin/python

import re, sys

words = []

for line in sys.stdin:
	line = re.sub("\n","",line) 
	line = re.sub("\s+"," ", line)
	words = line.split(' ')
	words = sorted(words)

	print (' '.join(map(str,words)))
	